packages/web3-shared/base/src/specs/index.ts

```diff
@@ -1360,6 +1363,19 @@ export interface HubState<
     getHub?: (options: Web3HubOptions) => Web3Hub
 }
 
+/** To create a writable storage, you need to pass a signer as parameter */
+declare function CreateNextIDStorage(
```

Comment:
What about moving them to the NextID provider API?

Reply:
This web3-shared package should not depend on the NextID provider.